nwbinspector/load_checks.py

```diff
@@ -0,0 +1,8 @@
+"""Authors: Cody Baker and Ben Dichter."""
+from .utils import default_checks
+from .check_datasets import check_dataset_compression
+from .check_time_series import check_regular_timestamps, check_data_orientation
+
+
+def load_checks(checks=default_checks):
+    return default_checks
```

Comment:
@bendichter This bit could also very well just go into `__init__.py`, what do you think?

Reply:
seems fine to me